packages/components/psammead-media-indicator/README.md

```diff
@@ -32,19 +29,14 @@ The `MediaIndicator` component provides a 'play', 'audio' or 'camera' icon as we
 
 The typical use-case of this component is on top of images within promos for articles that contains a video asset at the top of the page. It indicates to the user that the link is to a video and how long the video is in duration.
 
-For top story promos, we should pass the `topStory` prop to the `Media Indicator` to keep the same padding, otherwise this will be modified under 400px.
-
 Top story promos can also have related stories links called `Index Alsos`. When an Index Also item is a video, audio or photogallery, an icon should be displayed with the content using the `MediaIndicator` component passing the `isInline` prop.
 
+We expect users to add their own spacing around this component where relevant.
```

Comment:
Maybe it's worth rephrasing this a bit since "users" sounds like users of the website, not developers